apis/v1beta1/httproute_types.go

```diff
@@ -207,8 +207,10 @@ type HTTPRouteRule struct {
 	// All filters are expected to be compatible with each other except for the
 	// URLRewrite and RequestRedirect filters, which may not be combined. If an
 	// implementation can not support other combinations of filters, they must clearly
-	// document that limitation. In all cases where incompatible or unsupported
-	// filters are specified, implementations MUST add a warning condition to status.
+	// document that limitation. In cases where incompatible or unsupported
+	// filters are specified and cause the `Accepted` condition to be set to status
```

Comment:
this section `cause the Accepted condition to be set to status False` is because you might have a partially valid route, in that case it would still be `Accepted=True` so doesn't really make sense to set the `IncompatibleFilters` reason

Reply:
Yeah, I ran into the same problem in https://github.com/kubernetes-sigs/gateway-api/pull/2155 where it would be _really_ useful to be able to communicate "something's not quite right, but we're still able to at least partially reconcile this route". In that case it was if the timeout specified was more precise (ie ms or ns) than the underlying implementation could support.